src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs

```diff
@@ -129,6 +129,18 @@ public ActionResult Archive(int tournamentId)
             return new EmptyResult();
         }
 
+        /// <summary>
+        /// Unarchive current Tournament
+        /// </summary>
+        /// <param name="tournamentId">Tournament id</param>
+        /// <returns>Index View of Tournament</returns>        
+        [HttpPost]
+        public ActionResult UnArchive(int tournamentId)
```

Comment:
How this differs from line 296?

Reply:
One for actionlink which we use at details view 
another for actionlink which we use at archived views 

and i do not know how to use one method for that two actionlink

Reply:
Why they have to be different? Why they have different implementation?